app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -223,12 +270,12 @@ protected void getUserInfo(final AccessToken access_token) {
                     @Override
                     public void onCompleted(JSONObject object, GraphResponse response) {
                         try {
-                            final String id = object.getString("id");
-                            final String email = object.getString("email");
-                            final String name = object.getString("name");
-                            final String profPicUrl = object.getJSONObject("picture").getJSONObject("data").optString("url");
+                            final String id = object.getString(ID);
+                            final String email = object.getString(EMAIL);
+                            final String name = object.getString(NAME);
+                            final String profPicUrl = object.getJSONObject(PICTURE).getJSONObject(DATA).optString(URL);
```

Comment:
Same as above. what happens if these values return null? It would crash the app right?

Reply:
Some of these (like url) are nullable, but id, email, and name are not. All fb users have id and name, but not necessarily email so I will check that.

Reply:
I will make email a nullable field in newInstance for now since we don't have plans to use it yet.